src/aiida_quantumespresso/workflows/bands/base.py

```diff
@@ -28,7 +28,7 @@ def define(cls, spec):
             cls.results,
         )
         spec.exit_code(300, 'ERROR_UNRECOVERABLE_FAILURE',
-            message='The calculation failed with an unrecoverable error.')
+            message='The calculation failed with an unrecoverable error. [deprecated]')
```

Comment:
I'd put the `[deprecated]` at the front of the message, so it's more clear which exit codes are deprecated when a user runs `verdi plugin list`.